### ISSUE TITLE
Failure to support deeply nested structured datatypes by REST API (in SGv2)

### ISSUE THREAD

**Issue by tatu-at-datastax** [stargate/stargate#2061]
Data stored in a table defined as:

```
CREATE TABLE test_keyspace.conv_timeline (
  conv_id bigint PRIMARY KEY,
  data map<text, frozen<list<frozen<tuple<double, double>>>>>
)
```

seems to trigger a failure with stack trace like:

```
java.lang.IllegalArgumentException: Internal problem: failed to create converter for column #2/2 ('data'): Invalid/unsupported ColumnSpec TypeSpec SPEC_NOT_SET for column 'data'

at io.stargate.sgv2.restsvc.grpc.BridgeProtoValueConverters.fromProtoConverter(BridgeProtoValueConverters.java:41)
at io.stargate.sgv2.restsvc.resources.ResourceBase.convertRows(ResourceBase.java:155)
at io.stargate.sgv2.restsvc.resources.ResourceBase.toHttpResponse(ResourceBase.java:148)
at io.stargate.sgv2.restsvc.resources.ResourceBase.fetchRows(ResourceBase.java:140)
at io.stargate.sgv2.restsvc.resources.Sgv2RowsResourceImpl.getAllRows(Sgv2RowsResourceImpl.java:195)
```

with Stargate V2 REST API implementation. Same likely works in Stargate V1 REST; would make sense to add test for both if feasible.

**Comment by jmon-ds** [comment]
A user tried testing again recently and got the same error on 09/07/2022.  He also showed the same error on many other tables too with diff data while when he queried the same data via CQL it worked just fine. He confirmed the same queries worked for Java applications. It's just the REST API (both swagger & curl calls failed).

We (two DataStax engineers) had tried but can't reproduce it in our local environments.

**Comment by tatu-at-datastax** [comment]
yes, this would only be for REST service. As to local environments, this would affect builds from `v2.0.0`, not `master` (Stargate V2 vs V1), so that could explain the difference.

**Comment by tatu-at-datastax** [comment]
Found the issue, verifying the fix (see #2063).

**Comment by tatu-at-datastax** [comment]
Fix merged after successful test runs, closing.
